test/fail/ok/bounds-mismatch.tc.ok

```diff
@@ -35,34 +65,74 @@ bounds-mismatch.mo:75.21-75.22: type error [M0096], expression of type
   <X>X -> X
 cannot produce expected type
   <Y <: Nat>Y -> Y
+because: Incompatible function types: 
+  <X>X -> X
+ does not match 
+  <Y <: Nat>Y -> Y
+ in top level
 bounds-mismatch.mo:81.21-81.22: type error [M0096], expression of type
   <X <: Int, Y <: {}>X -> Y
 cannot produce expected type
   <X <: Int, Y <: {x : Nat}>X -> Y
+because: Incompatible function types: 
```

Comment:
would be nice if we could pinpoint the incompatible bound.

Reply:
true! I'll add func types to the redundant check